spec/integration/logging_spec.rb

```diff
@@ -50,6 +50,13 @@
     expect(result[0]['result']['_output'].strip).to match(/hi there/)
   end
 
+  it 'logs description when included' do
```

Comment:
I could make this replace the similar test above to avoid too much ssh integration testing.